src/main/java/com/spikes2212/command/LEDs.java

```diff
@@ -0,0 +1,94 @@
+package com.spikes2212.command;
+
+import com.spikes2212.dashboard.Namespace;
+import com.spikes2212.dashboard.RootNamespace;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+
+import java.awt.*;
+
+/**
+ * Class that gives more control over LEDs.
+ *
+ * @author Camellia Lami
+ * @see DashboardedSubsystem
+ */
+public class LEDs extends DashboardedSubsystem {
```

Comment:
1. Maybe change the name to `AddressableLEDWrapper`
2. Why does it extend `DashboardedSubsystem`?